build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -108,7 +108,7 @@ def execute_cmd(args: Sequence[Any],
     raise exc
 
 
-def execute_cmd_and_get_output(args: Sequence[Any],
+def execute_cmd_and_get_stdout(args: Sequence[Any],
```

Comment:
This can be changed to call `execute_cmd_and_get_output` and drop the stderr?

Reply:
That should be fine as long as we only care about the contents of `stderr` for debugging purposes when `subprocess` catches a non-zero exit code (as `execute_cmd` will only log it to the console in that case)!

Reply:
The difference is that if not captured stderr will be streamed to the console, which makes a significant difference for long-running commands. Also, not all debugging necessarily happens in response to non-zero exit codes. OTOH, this has the advantage of avoiding log spam if stderr would otherwise be noisy. I haven't surveyed all the commands being used with this function, but this is a change in behavior

Reply:
Ah, good point, I missed this comment before the approval. I'll let @phoenix-meadowlark decide if we want to revert this change.

Reply:
Oh no worries 🙂 I left it after this was merged (like an hour ago). I think reverting is probably unnecessary (unless people are complaining that all their logs disappeared), but I do think we want to leave the option to stream stderr somewhere

Reply:
Happy to write a follow-up that reverts `execute_cmd_and_get_stdout` to its previous behavior if that's desired! Piping it doesn't provide any additional information, and if a caller wants that they can use `execute_cmd_and_get_output` or pass the appropriate kwarg.